setup.py

```diff
@@ -0,0 +1,47 @@
+from setuptools import setup, find_packages
+import os
+import sys
+
+# Version
+BASE_DIR = os.path.dirname(os.path.realpath(__file__))
+VERSION_FILE = os.path.join(BASE_DIR, 'datasets', '_version.py')
+with open(VERSION_FILE) as fp:
+    exec(fp.read())
+
+with open("README.md", "r") as f:
+    long_description = f.read()
+
+setup(
+    name = "pydicom-data",
+    packages = find_packages(),
+    include_package_data = True,
+    version = __version__,
+    zip_safe = False,
+    description = "",
+    long_description = long_description,
+    long_description_content_type="text/x-rst",
+    author = "",
+    author_email = "FIXME",
```

Comment:
What would you like here @darcymason?

Reply:
Also, if I've jumped the gun here, just tell me

Reply:
I'm just catching up now, but I like the concepts...